### ISSUE TITLE
The output figure file is strange: there is a lot of blank area on the output figure.

### ISSUE THREAD

**Issue by geatpy-dev** [matplotlib/matplotlib#12376]
### Bug report

**Bug summary**

**Code for reproduction**

```python
# 
plt.savefig('Photo.png', dpi=600, bbox_inches='tight')
#
```

**Actual outcome**

![image](https://user-images.githubusercontent.com/42562919/46396899-43cb9b80-c723-11e8-8945-b20be5c52014.png)
```
# If applicable, paste the console output here
# The output figure can be seen as below:

#
```

**Expected outcome**

**Matplotlib version**

  * Operating system: windows10 x64
  * Matplotlib version: 3.0.0
  * Matplotlib backend (`print(matplotlib.get_backend())`):
  * Python version: 3.5、3.6
  * Jupyter version (if applicable):
  * Other libraries: 

The output figure file is strange: there is a lot of blank area on the output figure.

**Comment by geatpy-dev** [comment]
The output figure is normal if I used matplotlib v2.2.3, but having problem in v3.0.0, please help me, thanks!

**Comment by QuLogic** [comment]
You will need to provide more code than just a single final line for us to be able to say anything.

**Comment by geatpy-dev** [comment]
> You will need to provide more code than just a single final line for us to be able to say anything.

Thank you very much. I found a solution just now: just change the code from "plt.savefig('Photo.png', dpi=600, bbox_inches='tight')" to "plt.savefig('Photo.png', dpi=600)", and now the output figure is normal.
![image](https://user-images.githubusercontent.com/42562919/46397623-3dd6ba00-c725-11e8-9c17-b6500193a9cf.png)

But I have no idea why aftering deleting the code "bbox_inches='tight'" and the output figure is normal.
The whloe codes are show as below:
![image](https://user-images.githubusercontent.com/42562919/46397759-a7ef5f00-c725-11e8-95ac-1588d6ca47c3.png)

**Comment by dstansby** [comment]
Is there any chance you could provide a complete, minimal, self contained example that we can run to try and reproduce the issue? Otherwise it is hard to see what is going on to cause the issue.

**Comment by jklymak** [comment]
This is almost certainly a duplicate of #12239 and closed by #12241.

Please feel free to recommend if this is not the case.
